api/api/spec/changes.md

```diff
@@ -19,6 +19,9 @@
 
 ### GET /agents
 * Parameter **status** renamed to **agent_status**
+* Parameter **os.name** renamed to **os_name**
```

Comment:
This change will require changes in the framework. More precisely, here:
https://github.com/wazuh/wazuh/blob/8a34d85662154d324cd4c01455e205dc34367f04/framework/wazuh/agent.py#L116
https://github.com/wazuh/wazuh/blob/8a34d85662154d324cd4c01455e205dc34367f04/framework/wazuh/agent.py#L130
https://github.com/wazuh/wazuh/blob/8a34d85662154d324cd4c01455e205dc34367f04/framework/wazuh/agent.py#L334
Parameter `'.'` must be changed to `'_'`.

Reply:
Totally aggre. @JmZero please consider this since you are in charge of #2835

Reply:
It's not necessary change the framework. @crd1985 solved this using a dict:
https://github.com/wazuh/wazuh/blob/9d1f5c510d86e67ae5b730d8a704de4b02b21526/api/api/controllers/agents_controller.py#L105-L124